template/elm-package.json

```diff
@@ -10,7 +10,9 @@
     "exposed-modules": [],
     "dependencies": {
         "elm-lang/core": "5.0.0 <= v < 6.0.0",
-        "elm-lang/html": "2.0.0 <= v < 3.0.0"
+        "elm-lang/html": "2.0.0 <= v < 3.0.0",
+        "rtfeldman/elm-css": "8.2.0 <= v < 9.0.0",
```

Comment:
I'm afraid `elm-css` is supposed to be purely optional, the template should be kept in its initial state.